cmake/common.cmake

```diff
@@ -70,8 +80,10 @@ macro(nbl_create_executable_project _EXTRA_SOURCES _EXTRA_OPTIONS _EXTRA_INCLUDE
 	if(NBL_STATIC_BUILD)
 		add_dependencies(${EXECUTABLE_NAME} Nabla)
 	else()
-		add_dependencies(${EXECUTABLE_NAME}_with_config Nabla_with_manifest)
-		#target_link_options(${EXECUTABLE_NAME} PRIVATE "/manifestdependency:\"type='win32' name='devshgraphicsprogramming.nabla' version='1.2.3.4' processorArchitecture='x86' language='*'\"")
+		if(NBL_MSVC_GENERATE_APPLICATION_CONFIGS)
+			add_dependencies(${EXECUTABLE_NAME}_with_config Nabla_with_manifest)
+			#target_link_options(${EXECUTABLE_NAME} PRIVATE "/manifestdependency:\"type='win32' name='devshgraphicsprogramming.nabla' version='1.2.3.4' processorArchitecture='x86' language='*'\"")
+		endif()
```

Comment:
I guess we don't need manifests anymore then?

Reply:
we won't use it for sure now, but I would rather leave it for now because it's ready to use if they change `probing` policy

Reply:
removed